src/main/java/org/miracum/streams/ume/obdstofhir/mapper/ObdsToFhirMapper.java

```diff
@@ -18,13 +18,25 @@
 import org.miracum.streams.ume.obdstofhir.model.MeldungExportList;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.util.StringUtils;
 
 public abstract class ObdsToFhirMapper {
   protected final FhirProperties fhirProperties;
+  static Pattern localPatientIdPattern = Pattern.compile("[^0]\\d{8}");
 
   private static final Logger log = LoggerFactory.getLogger(ObdsToFhirMapper.class);
 
+  @Value("${app.localPatientIdPattern:[^0]\\d{8}}")
```

Comment:
```suggestion
  @Value("${app.patient-id-pattern}")
```

Müsste reichen wenn der default in der application.yaml steht (auch für die tests?)

Reply:
@Value mit Setter Injection braucht einen Default-Wert.